container/container.go

```diff
@@ -461,14 +470,15 @@ func getResource(name string, taskInfo *mesos.TaskInfo) *mesos.Resource {
 	return nil
 }
 
-// Prefix used to name Docker containers in order to distinguish those
-// created by Mesos from those created manually.
-const DockerNamePrefix = "mesos-"
-
 func GetContainerName(taskId *mesos.TaskID) string {
 	// unique uuid based on the TaskId
 	containerUUID := uuid.NewSHA1(uuid.NIL, []byte(*taskId.Value))
```

Comment:
What's the point in not just using the TaskID here instead of the UUID? I forget. Seems like a simple solution.

Reply:
Not sure, I was curious about this as well, but I assumed it was done on purpose and left it as it was. @felixgborrego is very worried that merging these changes will break the cluster, so we ended up postponing it for now. Let me know if you try it :)

Reply:
While going through this code, I think I see now why we computed the SHA of the Task ID, since it can contain dots, which are the `dockerNameSeparator`, so Mesos will get confused. I don't think I'll get a chance to test this properly, but if we'll want to merge these changes, we'll need to remove the dots from `taskId.Value`.